packages/frontend/src/components/dialogs/ViewGroup.tsx

```diff
@@ -136,6 +137,14 @@ function ViewGroupInner(
     setGroupImage,
   } = useGroup(accountId, chat)
 
+  const [pastContacts, setPastContacts] = useState([])
+
+  BackendRemote.rpc
+    .getContactsByIds(accountId, group.pastContactIds)
+    .then(pastContacts => {
+      setPastContacts(Object.values(pastContacts))
+    })
```

Comment:
This needs to be in `useEffect` in order to not call this on every re-render.
Perhaps also with `outdated` variable, as in https://github.com/deltachat/deltachat-desktop/pull/4500/files.